src/applications/check-in/utils/appointment/index.js

```diff
@@ -417,7 +417,7 @@ const getCheckinableAppointments = appointments => {
 const convertAppointments = appointments => {
   return appointments.map(appointment => ({
     id: appointment.id,
-    facility: appointment.attributes.location,
+    facility: appointment.attributes.location, // This is going away. We will switch to use facilityName
```

Comment:
Should we just make the change now?

Reply:
I think we should wait for the hand off from BE because other things might change there.